src/lib/interfaces/profile-deployment.ts

```diff
@@ -47,11 +47,25 @@ interface ContractOptions {
   deployProxy?: boolean;
   libAddress?: string;
 }
-export interface ContractDeploymentOptions {
+
+export interface ContractDeploymentOptionsReactive {
   version?: string;
-  deployReactive?: boolean;
+  deployReactive: true;
   uploadOptions?: UploadOptions;
   ERC725Account?: ContractOptions;
   KeyManager?: ContractOptions;
   UniversalReceiverDelegate?: ContractOptions;
 }
+
+export interface ContractDeploymentOptionsNonReactive {
+  version?: string;
+  deployReactive?: false;
+  uploadOptions?: UploadOptions;
+  ERC725Account?: ContractOptions;
+  KeyManager?: ContractOptions;
+  UniversalReceiverDelegate?: ContractOptions;
+}
+
```

Comment:
same here, maybe you can use a `Base` interface to extend from for the 2 interfaces.